start-a-node.md

```diff
@@ -37,6 +37,7 @@ Flag | Description
 `--join` | The address for connecting the node to an existing cluster. When starting the first node, leave this flag out. When starting subsequent nodes, set this flag to the address of any existing node. Optionally, you can specify the addresses of multiple existing nodes as a comma-separated list.
 `--key` | The path to the [node key](create-security-certificates.html) protecting the node certificate. This flag is required to start a secure node. 
 `--port`<br>`-p` | The port to listen on for internal and client communication. <br><br>**Default:** 26257
+`--raft-tick-interval` | CockroachDB uses the [Raft consensus algorithm](https://raft.github.io/) to replicate data consistently according to your [replication zone configuration](configure-replication-zones.html). For each replica group, an elected leader heartbeats its followers and keeps their logs replicated. When followers fail to receive heartbeats, a new leader is elected. <br><br>This flag sets the interval at which the replica leader heartbeats followers. For high-latency environments, you can adjust this interval to prevent leader elections, during which the operations on affected replicas stop.<br><br>**Default:** 100ms 
```

Comment:
This should say specifically that it should be set to a value greater than the average network latency between your nodes. Also if this flag is set anywhere, it should be set to the same value on all nodes.